### DISCUSSION TITLE
"Verrückt" - German Spawn Area Teaser

### DISCUSSION THREAD

**Discussion by MotoLegacy** [nzp-team/nzportable#527] [Announcements]
https://www.youtube.com/watch?v=c6W4jW0kXoI

Feel free to inquire below.

**Comment by DerpedCrusader** [comment]
👀👀

**Comment by bignoobus** [comment]
Looking fancy smancy if i do say so myself 👍

**Comment by ToonLink0001** [comment]
when will this drop?

**Reply by Peter0x44** [reply]
When it's ready.

**Comment by Useless3** [comment]
Is this going to be released soon?

**Reply by katniny** [reply]
<img width="499" height="469" alt="image" src="https://github.com/user-attachments/assets/f3fbe487-7e4b-4540-89e7-edec87c7d494" />

**Reply by Useless3** [reply]
It's been like a year since that was posted, I just wanted an update.

**Reply by M0THRAK** [reply]
It'll be done when it's ready, don't worry.

**Reply by katniny** [reply]
yes
nearly every update makes nz:p better to prepare for verruckt
everyone just has to be patient

**Comment by ProfessorZIRE** [comment]
3 questions
1. Will there be the notorious verrukt super sprinters?
2. Will there be the split co-op spawns? 
3. Are u guys doing shi no numa next?
If u are not going to reveal this information, it is fine, I'm just curious (I've spent so much time on WaW zombies so I just want all 4 maps to be in nzp)

**Reply by MotoLegacy** [reply]
1. yea
2. yea
3. yea

**Reply by ProfessorZIRE** [reply]
LET'S GOOOOOOOO
